scripts/firerouter_dhclient_ip_change

```diff
@@ -12,7 +12,7 @@ case "$reason" in
     fi
     ;;
   TIMEOUT)
-    if [ "$exit_status" == "0" ]; then
+    if [ "$exit_status" = "0" ]; then
```

Comment:
If we are sure that `exit_status` has been defined, it is better to test with `-eq`

`if [ $exit_status -eq 0 ]; then ...`